contrib/babelfishpg_tsql/src/guc.c

```diff
@@ -1207,6 +1212,8 @@ int			escape_hatch_rowversion = EH_STRICT;
 int			escape_hatch_showplan_all = EH_STRICT;
 int			escape_hatch_checkpoint = EH_IGNORE;
 int			escape_hatch_set_transaction_isolation_level = EH_STRICT;
+int			pltsql_isolation_level_repeatable_read = ISOLATION_OFF;
+int 		pltsql_isolation_level_serializable = ISOLATION_OFF;
```

Comment:
indent

Reply:
Not sure why this it is off, both are 2 tabs & 1 space